sass/tji/_variables.scss

```diff
@@ -29,4 +29,27 @@ $tji-line-height-md: 3.6rem;
 $tji-line-height-lg: 4.6rem;
 $tji-line-height-xl: 5.4rem;
 
-$tji-chart-legend-size: 1rem;
\ No newline at end of file
+$tji-chart-legend-size: 1rem;
+
+/* classes for extension */
+
+@mixin bootstrap-button() {
+  border-radius: .25rem;
+  cursor: pointer;
+  display: inline-block;
+}
+
+%bootstrap-button-secondary {
+  @include bootstrap-button();
+  color: white;
+  background-color: #6c757d;
+  border-color: ##6c757d;
```

Comment:
Everett, there is an extra "#" on line 46 that needs to be removed. Also, do we want to use one of our colors already set as a variable for the button background?

Reply:
Good catch. Also, updated the whole shebang to use our colors/variables.